flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/LeaderChangeClusterComponentsTest.java

```diff
@@ -135,6 +138,8 @@ public void testReelectionOfJobMaster() throws Exception {
 
 		submissionFuture.get();
 
+		CommonTestUtils.waitUntilJobManagerIsInitialized(() -> miniCluster.getJobStatus(jobId).get());
```

Comment:
Same here?

Reply:
The next statement in the test is to revoke the JobMasterLeadership.
This revocation is only possible if we have leadership, which depends on the job being initialized.